### PR TITLE
Add note to Capella validator guide about BLS to execution change

### PR DESCRIPTION
**Pull request by ralexstokes**
^

### CODE REVIEW

File: specs/capella/validator.md [hunk 1]
```diff
@@ -111,3 +113,38 @@ def prepare_execution_payload(state: BeaconState,
 ##### BLS to execution changes
 
 Up to `MAX_BLS_TO_EXECUTION_CHANGES`, [`BLSToExecutionChange`](./beacon-chain.md#blstoexecutionchange) objects can be included in the `block`. The BLS to execution changes must satisfy the verification conditions found in [BLS to execution change processing](./beacon-chain.md#new-process_bls_to_execution_change).
+
+## Enabling validator withdrawals
+
+Stake belonging to validators that can safely be withdrawn from the beacon chain happens via an automatic process.
+
+There is one prerequisite task to perform before a validator can particpate in this automatic process:
```
Comment (hwwhww) [inline specs/capella/validator.md:121]:
```suggestion
There is one prerequisite task to perform before a validator can participate in this automatic process:
```

File: specs/capella/validator.md [hunk 2]
```diff
@@ -111,3 +113,38 @@ def prepare_execution_payload(state: BeaconState,
 ##### BLS to execution changes
 
 Up to `MAX_BLS_TO_EXECUTION_CHANGES`, [`BLSToExecutionChange`](./beacon-chain.md#blstoexecutionchange) objects can be included in the `block`. The BLS to execution changes must satisfy the verification conditions found in [BLS to execution change processing](./beacon-chain.md#new-process_bls_to_execution_change).
+
+## Enabling validator withdrawals
+
+Stake belonging to validators that can safely be withdrawn from the beacon chain happens via an automatic process.
```
Comment (hwwhww) [inline specs/capella/validator.md:119]:
```suggestion
Stake belonging to validators that can safely be withdrawn partially or fully from the beacon chain happens via an automatic process.
```

File: specs/capella/validator.md [hunk 3]
```diff
@@ -111,3 +113,38 @@ def prepare_execution_payload(state: BeaconState,
...
+This `message` contains the `validator_index` for the validator who wishes to change their credentials, the `from_bls_pubkey` containing the BLS public key corresponding to the **withdrawal BLS secret key** used to form the `BLS_WITHDRAWAL_PREFIX` withdrawal credential, and the `to_execution_address` specifying where the validator wants stake to be withdrawn to on the execution layer.
+
+**NOTE**: The withdrawal key pair used to construct the `BLS_WITHDRAWAL_PREFIX` withdrawal credential should be distinct from the signing key pair used to operate the validator under typical circumstances. Consult your validator deposit tooling documentation for further details if you are not aware of the difference.
+
+The `from_bls_pubkey` is compared against the existing withdrawal credential on-chain for the given `validator_index` so
+it is unlikely a validator will incorrectly specify this data; however, this message can only be issued once and is currently
+irreversible once on-chain so take care with the `to_execution_address`.
+
+Next, the validator signs the assembled `BLSToExecutionChange` `message` with the **withdrawal BLS secret key** and this
```
Comment (hwwhww) [inline specs/capella/validator.md:144]:
```suggestion
Next, the validator signs the assembled `message: BLSToExecutionChange` with the **withdrawal BLS secret key** and this
```

File: specs/capella/validator.md [hunk 4]
```diff
@@ -111,3 +113,38 @@ def prepare_execution_payload(state: BeaconState,
...
+`signature` is placed into a `SignedBLSToExecutionChange` message along with the inner `BLSToExecutionChange` `message`.
+Note that the `SignedBLSToExecutionChange` message should pass all of the validations in [`process_bls_to_execution_change`](./beacon-chain.md#new-process_bls_to_execution_change).
+
+The complete `SignedBLSToExecutionChange` message can be submitted to the consensus layer network. Once included on-chain,
+the withdrawal credential change takes effect. No further action is required for a validator to enter into the automated
```
Comment (hwwhww) [inline specs/capella/validator.md:149]: nitpicking: to do full withdrawal, the validator has to do voluntary exit first.

**Review by hwwhww — COMMENTED** [review]
a quick scan here.

we should also update p2p-interface.md about how to broadcast it.

**Comment by lucassaldanha** [comment]
> we should also update p2p-interface.md about how to broadcast it.

@hwwhww  `p2p-interface.md` has recently been updated with details on the new global topic for gossiping bls_to_execution_change messages. Is this what you were talking about?

**Comment by hwwhww** [comment]
@lucassaldanha ah you're right. sorry I missed it.

**Review by djrtwo — APPROVED** [review]
nice!

**Pull request merged**